Quote.go

```diff
@@ -0,0 +1,238 @@
+package arn
+
+import (
+	"errors"
+
+	"github.com/aerogo/nano"
+	"github.com/fatih/color"
+	"sort"
+)
+
+// Quote ...
+type Quote struct {
+	ID          string   `json:"id"`
+	Description string   `json:"description" editable:"true" type:"textarea"`
+	CharacterId string   `json:"characterId" editable:"true"`
+	AnimeId     string   `json:"animeId" editable:"true"`
+	Episode     int      `json:"episode" editable:"true"`
```

Comment:
This should probably be `EpisodeNumber` while `Episode()` could be the name of a potential method that fetches the actual episode.